ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-725] Reset necessary form attributes in income section

## Description of change
Spotted during qa, if a user changes their answer to the `Is your client’s income currently more than £12,475 a year before tax?` question, any responses previously recorded for conditionally routed questions should be reset. This affects any answers recorded to the following: 

1. Does your client have any income, savings or assets under a restraint or freezing order?
2. Does your client own their home, or any other land or property?
3. Does your client have any savings or investments?

## Link to relevant ticket
[CRIMAPP-725](https://dsdmoj.atlassian.net/browse/CRIMAPP-725)

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:

### After changes:

https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/dd3d843d-184d-44df-ba89-3de57c3d48d1


## How to manually test the feature

1. In Apply process an application and navigate to the Income section
2. For question 'Is your client’s income currently more than £12,475 a year before tax? answer NO
3. For question Does your client have any income, savings or assets under a restraint or freezing order? answer NO
4. For question Does your client own their home, or any other land or property? answer NO
5. For question Does your client have any savings or investments? answer YES
6. On the Income CYA page on the Income section the questions and answers are displayed as above
7. Use the Change link on CYA to go to the Is your client’s income currently more than £12,475 a year before tax? question. Change the answer to YES
8. Save and Continue. The form now routes to the Which of these payments does your client get? page. This is correct as the other 3 questions above no longer need answering.
9. On the Income CYA page the questions and answers to for step 3, 4, 5 above should have been reset and should no longer display


[CRIMAPP-725]: https://dsdmoj.atlassian.net/browse/CRIMAPP-725?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
